roles/nginx/tasks/install.yml

```diff
@@ -25,3 +25,11 @@
     - { src: "roles/nginx/templates/server.conf", dest: "/etc/nginx/" }
     - { src: "roles/nginx/templates/nginx.conf", dest: "/etc/nginx/" }
     - { src: 'roles/nginx/templates/ports.conf', dest: '/etc/{{ apache_service }}/' , mode: '0644' }
+    - { src: 'roles/nginx/templates/uwsgi.service', dest: '/etc/systemd/system/' , mode: '0644' }
```

Comment:
This will override the default uwsgi.service which parses /etc/uwsgi/apps-enabled/ for any .ini files out of the box which is used by https://github.com/iiab/iiab-admin-console/blob/master/roles/console/tasks/main.yml without needing to supply this service file that would limit uwsgi to only run admin-console.ini

Reply:
I did it the way you suggest first and could not get uwsgi to come up properly after reboot -- maybe because admin-console was using socket, and captive portal was using port. The docs suggest that the way to solve is with uwsgi emperor. I chose to kiss. 
I wanted to keep captive portal on a port for debugging simplicity -- see https://github.com/georgejhunt/iiab/blob/cap3.1/roles/captiveportal/README.md

Reply:
I'm missing something, admin-console.ini is installed to /etc/uwsgi/apps-enabled/ is supplied by iiab-admin-console and the stock uwsqi.service is enabled there. There is no uwsqi service enabled for nginx's role, it's been moved.